dts/riscv/riscv32-litex-vexriscv.dtsi

```diff
@@ -63,5 +63,13 @@
 			label = "eth0";
 			status = "disabled";
 		};
+		dna0: dna@e0003000 {
+			compatible = "litex,dna0";
+			/* DNA value is 47-bits long, register width is aligned to 64 */
+			reg = <0xe0003000 0x40>;
```

Comment:
I think we usually use the numbers of registers here. Not the numbers of bits. But maybe I'm wrong.

Reply:
I don't know if there is a strict definition of what the size means in Zephyr.
From what I see in other sections of this and other dtses it looks that the value should be the sum of widths of all registers expressed in *bytes*, so in this context: 0x20 (8 registers, each 4-byte).

I'll change it to be consistent and avoid further confusion.